tests/e2e/yamcs/realtimeData.e2e.spec.mjs

```diff
@@ -249,6 +251,101 @@ test.describe('Realtime telemetry displays', () => {
             }
         });
 
+        test.only('Open MCT does not drop telemetry when a burst of telemetry arrives', async ({ page }) => {
```

Comment:
test('Does not drop telemetry when a burst of telemetry greater than the buffer arrives')

Reply:
Good call, will do.